vega-tooltip/build.boot

```diff
@@ -17,6 +17,29 @@
        :scm         {:url "https://github.com/cljsjs/packages"}})
 
 (deftask package []
+  (task-options! push {:ensure-branch nil})
+  (comp
+    (download
+     :url (format "https://unpkg.com/vega-tooltip@%s/build/vega-tooltip.js" +lib-version+)
+     :checksum "9DA40C6DFB095BECD7DED0C1140306C7")
```

Comment:
By the way, instead of including these checksums here inline, you can add task `(validate)` as the final thing in the `package` task, after `(jar)`. Then Boot will automatically store the checksums in a separate file `boot-cljsjs-packages.edn` and ask you if you want to update when you're building the package with `boot package`. See my recent pull request #1680 for an example.

It's not necessary to change it, but it would make updating this package the next time a bit easier.